### ISSUE TITLE
Feature Request: Notification System - Email and / or Webhooks options for Chat application additions

### ISSUE THREAD

**Issue by bmcgonag** [cloudradar-monitoring/rport#499]
Story:
As an RPort Administrative user it's important for me to know when certain client machines are disconnected for too long, as this indicates something may be wrong with the client or the internet / network connection of the client; so I need a way to receive an automated notification from the RPort system when such an event occurs.

Follow up stories:
As an RPort Administrator, I need a way to set limits on resource usage based on monitoring, and when those limits are exceeded for a given amount of time, I need to receive an automated notification via the notification system. (example: Disk I/O too high for more than 5 minutes consecutively, or Disk space lower than X GB, all CPU at 100% for longer than 10 minutes consecutively, etc).

As an RPort Administrator, I want to receive notifications when certain commands or scripts are run by my users on a client or group of clients.

**Comment by thorstenkramm** [comment]
@bmcgonag Thank you for this request. Your points are valid and understandable. 

Nevertheless, I kindly ask you to withdraw this request. Create three separated requests on the [feature request discussion](https://github.com/cloudradar-monitoring/rport/discussions/categories/roadmap-feature-wishes-ideas) instead.  

Use clear subject, do not mix topics.
If topics and wishes are mixed in a single threat, it's hard to follow and others cannot up vote.
